### DISCUSSION TITLE
Wireless keyboard

### DISCUSSION THREAD

**Discussion by RedSparrowEU** [UzixLS/zx-sizif-512#53] [Q&A]
Hi,

Just googled first without clear results...
Is it possible to use Bluetooth keybord with Sizif?

Thanks,
Greg

**Comment by UzixLS** [comment]
Nope. Bluetooth module acts only as tape input interface.
But you can use wireless keyboard with PS/2 receiver.

**Reply by RedSparrowEU** [reply]
Thanks for the quick reply!

I have mainly BT keyboards. So BT pairing is rather required. Is it possible or needs some hacks? I mean with PS/2 BT
Anyway - the first choice if answer would be 'No' is this project: https://github.com/Hamberthm/esp32-bt2ps2/tree/main

BTW: As Wifi modul is already ESP32 maybe it is possible to expand it for BT keyboards?

**Reply by UzixLS** [reply]
Not sure about details about wireless PS/2 keyboards, I didn't deal with them.

Wifi module in Sizif is ESP8266, it doesn't have BT support.
